modules/Kconfig.lvgl

```diff
@@ -8,6 +8,27 @@ config LVGL
 
 if LVGL
 
+config LV_Z_BITS_PER_PIXEL
+	int
+
+config LV_Z_DOUBLE_VDB
+	bool
+
+config LV_Z_DPI
+	int
+
+config LV_Z_FULL_REFRESH
+	bool
+
+config LV_Z_VBD_CUSTOM_SECTION
+	bool
+
+config LV_Z_VDB_ALIGN
```

Comment:
looking to confirm that LV_Z_VDB_ALIGN and LV_Z_FULL_REFRESH were both intended to be here.  Asking as I don't see them from the shields entries.

Reply:
Yeah, they are in `boards/arm/mimxrt1060_evk/Kconfig.defconfig` (https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mimxrt1060_evk/Kconfig.defconfig#L69-L80)